### PR TITLE
6 & 7 complete

### PR DESCRIPTION
**Pull request by Connor-Quinny**
I had already done 6 from earlier

### CODE REVIEW

File: src/Components/ReviewCard.jsx [hunk 1]
```diff
@@ -5,12 +5,25 @@ import { useParams } from "react-router-dom"
 const ReviewCard = () => {
     const [review, setReview] = useState({})
     const {review_id} = useParams()
+    const [vote, setVotes] = useState(0)
 
     useEffect(() => {
```
Comment (mallionaire) [inline src/Components/ReviewCard.jsx:12]: Please extract your api calls into a `utils/api.js`

**Review by mallionaire — CHANGES_REQUESTED** [review]
Nice start! 
I've got. a few tips for you! 
You seem to be getting a few `Each child in a list` warnings, look into these, I suspect it might be the links causing the problem. 

You could use the axios `params` object to tidy up (and DRY up) your api requests if a category is provided. 

Also - the `users` and `games` tags in your nav bar seem unnecessary - will you ever be displaying a list of games, or was this something you copied from an example you've seen in the lecture? 

Also it would be nice to optimistically render the change to the vote, assuming that there will be a success from the api (if you change your internet speed in the network tab on devtools you can see that it takes some time for the vote change to come through)

**Review by Augs0 — CHANGES_REQUESTED** [review]
Hey Connor, can see lots of great changes made here. One more thing I'd like you to do is add a 404 page for when I go to a non-existent article. Currently, if I do that, I get a page with the labels and an undefined title which is not the best user experience

**Review by leekli — APPROVED** [review]
Well done Connor, you are free to merge this branch.

One point: If I go to a non-existant article in desktop view, I can see the 404 error message, but If I do it in mobile view I can't see the 404 message, just a blank page... I'd have a look at this in your next ticket but for now please do merge tickets 6 & 7 first. :)

**Pull request merged**